Processing/test_yOCTFindTissueSurface.m

```diff
@@ -14,12 +14,14 @@ function createDummyDataset(testCase)
             % Create a dummy dataset with a scattering interface
             testCase.simulatedSurfacePositionZ_pix = 500;
             speckleField = zeros(1024,100,200)+10; %z,x,y
-            speckleField(testCase.simulatedSurfacePositionZ_pix,:,:) = 1000;
-    
+            speckleField(testCase.simulatedSurfacePositionZ_pix, :, :) = 1000;
+        
             [interf, dim] = yOCTSimulateInterferogram(speckleField);
             [cpx, dim] = yOCTInterfToScanCpx(interf, dim);
             logMeanAbs_tmp = log(abs(cpx));
             testCase.logMeanAbs = logMeanAbs_tmp;
+            refSliceValue = testCase.logMeanAbs(testCase.simulatedSurfacePositionZ_pix, :, :);
```

Comment:
Move this to row 17. logMeansAbs shouldn't be changed after simulating the speckle field.